NAS-103292: Update cloud sync status description: (#1602)
freenas/freenas-docs: userguide/tasks.rst

   Cloud Sync Status

> The cloud sync :guilabel:`Status` indicates the state of the currently

Reviewer: "most recent" covers both.
```suggestion
The cloud sync :guilabel:`Status` indicates the state of the most recent
```

> running cloud sync or the result of the last cloud sync. Clicking the

Reviewer: ```suggestion
cloud sync. Clicking the
```

> :guilabel:`Status` entry shows the cloud sync task logs and includes an

Reviewer: Context is adequate.
```suggestion
:guilabel:`Status` entry shows the task logs and includes an
```

> option to download the task log.

Reviewer: ```suggestion
option to download them.
```
